src/util/properties.rs

```diff
@@ -183,6 +283,18 @@ impl UserProperties {
         buf.encode_u64(value).unwrap();
         self.encode(name, buf);
     }
+
+    // Format: | klen | k | v.size | v.offset |
+    pub fn encode_handles(&mut self, name: &str, handles: &BTreeMap<Vec<u8>, IndexHandle>) {
```

Comment:
Not really, it can be a common struct in the future. For example, we may add handles for rows count later.